tests/stubs/pama-imaging-stub.js

```diff
@@ -0,0 +1,102 @@
+const uuid = require('uuid');
+
+function getReasons(reasons) {
+  return reasons.map(r => (
+    {
+      coding: r.reasons.map(x => (
+        {
+          system: 'http://snomed.info/sct',
+          code: x.code,
+          display: x.display,
+        }
+      )),
+    }
+  ));
+}
+
+function getResources(
+  bundle, patientId,
+  intent = 'plan',
+  resourceType = 'ServiceRequest',
+  status = 'draft',
+) {
+  return bundle.map(x => (
+    {
+      resource: {
+        resourceType,
+        id: `example-${patientId}`,
```

Comment:
Is any of the testing contingent on this `id`? Might simplify it by returning a fixed value. (Though also I should note note: if `bundle` ever has >1 element we'll need to ensure these `id`s are unique.